docs/using-borges/workflow.md

```diff
@@ -0,0 +1,39 @@
+# Workflow
+
+## All components working together
+
+This is an example of several components working together. It's using a producer that generates jobs from the [rovers](https://github.com/src-d/rovers) mentions queue, a consumer that processes those jobs and a republish producer that moves back jobs with errors from buried queue to jobs queue.
```

Comment:
Let's put it into points:
- producer that generates jobs from the...
- consumer that processes those jobs and a republish producer...

Reply:
@kuba-- done